mlem/cli/apply.py

```diff
@@ -187,13 +190,17 @@ def _apply_remote(
 
 
 option_output = Option(
-    None, "-o", "--output", help="Where to store the outputs."
+    None,
+    "-o",
+    "--output",
+    help="Where to save inference results",
```

Comment:
Are model outputs always inferences? Idk. Maybe something more specific like "model's output files"? Up to you